FirebaseDynamicLinks/README.md

```diff
@@ -0,0 +1,25 @@
+# Firebase Dynamic Links SDK for iOS
+
+Firebase Dynamic Links are links that work the way you want, on multiple platforms, and whether or not your app is already installed.
+For more info : https://firebase.google.com/products/dynamic-links
```

Comment:
`For more info, see the [Firebase website](https://firebase.google.com/products/dynamic-links).`

Reply:
done